apps/prairielearn/src/models/course-request.sql

```diff
@@ -69,6 +69,12 @@ WITH
       INNER JOIN users AS u ON u.user_id = r.user_id
       LEFT JOIN users AS ua on ua.user_id = r.approved_by
       LEFT JOIN select_course_request_jobs AS j ON j.id = r.id
+    WHERE
+      (
+        $show_all != 'true'
+        AND r.approved_status NOT IN ('approved', 'denied')
+      )
+      OR ($show_all = 'true')
```

Comment:
As someone that just taught discrete math this is bothering me... :-)
```suggestion
    WHERE
      r.approved_status NOT IN ('approved', 'denied')
      OR $show_all = 'true'
```

Reply:
Even simpler than mine! I'd probably swap the order of the conditions to make it a little clearer, even though the end result is the same.

Reply:
Thanks for the help here! Updated to Jonatan's solution and swapped the order of the conditions.